examples/gallery/simple/clifford_interact.ipynb

```diff
@@ -15,7 +15,8 @@
     "from datashader import transfer_functions as tf\n",
     "from colorcet import palette_n\n",
     "\n",
-    "pn.extension()"
+    "pn.extension()\n",
+    "pn.config.sizing_mode=\"stretch_width\""
```

Comment:
It's a matter of preference but config options can be passed to the extension:

```suggestion
    "pn.extension(sizing_mode='stretch_width')"
```

Reply:
Thanks. I like that preference :-) I will update.

Reply:
@philippjfr . I have updated all Panel notebooks and test examples to use this terminology. It's nice and simple.